docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -306,6 +306,42 @@ renamed to better reflect its intended use.
 *Impact* +
 Assign users with the `kibana_user` role to the `kibana_admin` role.
 Discontinue use of the `kibana_user` role.
+====
+
+[discrete]
+[[saml-realm-nameid-changes]]
+===== The default value of `nameid_format` setting has been removed.
+
+.The `nameid_format` setting of SAML realms has no default value.
+[%collapsible]
+====
+*Details* +
+Identity Providers can be either statically configured to release a `NameID`
+with a specific format, or they can be configured to try to conform with the
+requirements of the SP. The SP declares its requirements as part of the
+Authentication Request, using an element which is called the `NameIDPolicy`
+For the {es} SAML realm, `NameIDPolicy` is controlled by `nameid_format` in
+SAML realm settings.
```

Comment:
I tried to streamline this and remove some passive voice. I'd also ensure we use acronyms like IdPs and SPs consistently.

I'm not as familiar with the audience here, but I wondered if we needed any of the context before the last sentence. Would the audience already be familiar with these concepts?
```suggestion
In SAML, Identity Providers (IdPs) either release a `NameID` or attempt to
conform with the requirements of a Service Provider (SP). The SP declares its
requirements in the `NameIDPolicy` of an authentication request. In {es}, the
`nameid_format` SAML realm setting controls the `NameIDPolicy`.
```